### ISSUE TITLE
To Dos: Boosted BBB IMPRS 2019-2020

### ISSUE THREAD

**Issue by TCLamnidis** [BareBonesBash/barebonesbash.github.io#5]
Thiseas:
 - [x] Move housekeeping before script session.

Aida:
 - [x] Few more slides explaining how scripting works.

James:
 - [x] Move conditionals section to introduce them before scripting session.
 - [x] Update conditionals to new structure
- [x] Add gifs. Final sections especially have a lot of space for them.

General:
 - [x] Add `||` and `&&` to conditionals. Used in scripting (helptext) but not introduced. - James
 - [x] Explain in plain english what each regex does. Still not sure wtf `grep '^[rb]\+' regex.txt` does. - Aida
  - [x] Maybe explain a why `\` escapes but `\+` does the opposite, instead treatinf `+` as a regex instead of character. - aida
 - [x] Final FIX: `regex.txt` in presentation is formatted badly. Sth to do with Atom? - James
- [x] Add helptext to `data/Script_example.sh`. Currently missing conditionals.
- [x] Test final `data/Script_example.sh`

**Comment by jfy133** [comment]
- [x] Review regex plain-english
- [x] Review `\+` explanation solution

**Comment by jfy133** [comment]
- [ ] run throughs! (just James to go through text)
- [ ] someone other than Thiseas to check script

**Comment by TCLamnidis** [comment]
- [x] While loop psudocodes. Should they be consistent? currently it changes.

**Comment by aidaanva** [comment]
In slide comparing rev+cut vs paramater expansion:

- [x] Make clear you run only the second block of code

In while loops:

- [x] Specify that the examples are pseudocode and must not be run

- [x] Include how to kill a loop that has gone wrong

In paste:

- [x] Add a disclaimer: it only works when both columns are in the same order

Final slide:

- [x] Make summary slides

**Comment by jfy133** [comment]
Write forms/prepare email/dates

- In person or online?
- In person - limit no. of people?
- Allow 'newcomers' (but no IMPRS points)?
- Dates?
- What to include in forms?
- Who communicates with Stephan re: cluster?

**Comment by jfy133** [comment]
- [ ] Transfer to text walkthrough
- [x] Test online systems (when decided)
- [x] Book rooms (when decided)
- [x] Setup and Check login enviroment is OK 
- [x] Assign speakers slides
- [x] James ask to hold onto 32node server
- [x] Check for placeholders

**Comment by TCLamnidis** [comment]
- [x] tested online systems (Big Blue Button side rooms)
- [x] Book Oval Office for us three once reminder email is sent out. Meeting will be online.
- [x] Update login section to James' server
- [ ] Check that everyone is happy with their slides

**Comment by TCLamnidis** [comment]
Placeholder exists but is hidden in presentation. Should add TOC from the walkthrough when that is ready.

**Comment by jfy133** [comment]
Note: before session and booting up server, ensure to `sudo apt install rename`!
